docs/sources/tasks/collect-logs-in-kubernetes.md

```diff
@@ -0,0 +1,323 @@
+
+1. Add the following `loki.write` component to your configuration file.
+
+   ```alloy
+   loki.write "<LABEL>" {
+     endpoint {
+       url = "<LOKI_URL>"
+     }
+   }
+   ```
+
+   Replace the following:
+
+   - _`<LABEL>`_: The label for the component, such as `default`.
+     The label you use must be unique across all `loki.write` components in the same configuration file.
+   - _`<LOKI_URL>`_ The full URL of the Loki endpoint where logs will be sent, such as `https://logs-us-central1.grafana.net/loki/api/v1/push`.
+
+1. If your endpoint requires basic authentication, paste the following inside the `endpoint` block.
+
+   ```alloy
+   basic_auth {
+     username = "<USERNAME>"
+     password = "<PASSWORD>"
+   }
+   ```
+
+   Replace the following:
+
+   - _`<USERNAME>`_: The basic authentication username.
+   - _`<PASSWORD>`_: The basic authentication password or API key.
+
+1. If you have more than one endpoint to write logs to, repeat the `endpoint` block for additional endpoints.
+
+The following example demonstrates configuring `loki.write` with multiple endpoints and mixed usage of basic authentication, and a `loki.source.file` component which forwards logs to it.
+
+```alloy
+loki.write "default" {
+  endpoint {
+    url = "http://localhost:3100/loki/api/v1/push"
+  }
+
+  endpoint {
+    url = "https://logs-us-central1.grafana.net/loki/api/v1/push"
+
+    // Get basic authentication based on environment variables.
+    basic_auth {
+      username = env("<REMOTE_WRITE_USERNAME>")
+      password = env("<REMOTE_WRITE_PASSWORD>")
+    }
+  }
+}
+
+loki.source.file "example" {
+  // Collect logs from the default listen address.
+  targets = [
+    {__path__ = "/tmp/foo.txt", "color" = "pink"},
+    {__path__ = "/tmp/bar.txt", "color" = "blue"},
+    {__path__ = "/tmp/baz.txt", "color" = "grey"},
+  ]
+
+  forward_to = [loki.write.default.receiver]
+}
+```
+
+For more information on configuring logs delivery, refer to [loki.write][].
+
+## Collect logs from Kubernetes
+
+{{< param "PRODUCT_NAME" >}} can be configured to collect all kind of logs from Kubernetes:
+
+1. System logs
+1. Pods logs
+1. Kubernetes Events
+
+Thanks to the component architecture, you can follow one or all of the next sections to get those logs. Once you have followed the [Configure logs delivery](#configure-logs-delivery) to ensure collected logs can be written somewhere, jump to the relevant sections.
+
+### System logs
+
+To get the system logs, you should use the following components:
+1. [local.file_match][]: Discovers files on the local filesystem
+1. [loki.source.file][]: Reads log entries from files
+1. [loki.write][]: Send logs to the Loki endpoint. You should have configured it in the [Configure logs delivery](#configure-logs-delivery) section
+
+Here is an example using those stages.
+
+```alloy
+// local.file_match discovers files on the local filesystem using glob patterns and the doublestar library. Expose an array of file path.
```

Comment:
I'm not sure what the intent of ` Expose an array of file path.` is here. Is this an instruction? or is there something missing from the sentence?

Reply:
I think the intent is to say that the component doesn't act, it just export a list of files for the next components to use.

Reply:
```suggestion
// local.file_match discovers files on the local filesystem using glob patterns and the doublestar library. It returns an array of file paths.
```

Is this still accurate then?